File: program/program/convert-miles-to-meters/ConvertMilesToMeters.java
import java.util.*;
import java.io.*;
import java.text.*;
// import java.lang;
public class Main
{
	public static void main(String[] args) {
	    DecimalFormat df = new DecimalFormat("0.00");
	    Scanner sc=new Scanner(System.in);
	    int miles=sc.nextInt();
		System.out.println(df.format(miles*1609.344));
	}
}

File: program/convert-miles-to-meters/ConvertMilesToMeters.kt
import java.util.Scanner

fun main() {
    print("Enter Miles : ")
    println()
    var miles = readLine()

    println("$miles miles in meters : " + ConvertMilesToMeters(miles!!.toInt()))

}

private fun ConvertMilesToMeters(miles: Int): Double {
    val meter = miles*1609.344;

    return  meter
}